types/mongoose/index.d.ts

```diff
@@ -3318,6 +3325,7 @@ declare module "mongoose" {
     where(path: string, val?: any): Query<any> & QueryHelpers;
   }
 
+  class Document {}
```

Comment:
as of the current mongoose 5.7, Document is an instantiable class. Maybe there is another way of writing this so that calls to `new Document` or `instanceof Document` will be allowed?

Reply:
The line below can be changed to be a class instead of an interface.

Reply:
Any ideas to fix this without modifying too much existing semantics? I think letting TS to merge the class and interface is simple enough to make it export a class.